arm-sql/2015-05-01/swagger/2015-05-01-all.json

```diff
@@ -0,0 +1,1914 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2015-05-01-preview",
```

Comment:
SQLServer create and delete should be long running operations in RP, as this is a preview version of APIs, this can be fixed in this version